component-interaction/src/app/user-list/user-list.component.html

```diff
@@ -1,5 +1,4 @@
 <button class="succes" (click)="handleAddUserClick()">Insert user!</button>
 <button *ngIf="selectedUserIds.length" (click)="handleDeleteSelectedUsersClick()">Delete users</button>
-
-<user-details *ngFor="let user of users | sortUsers: TaskStatus.ASC" [user]="user"
+<user-details *ngFor="let user of (users | sortUsers: TaskStatus.ASC)" [user]="user"
```

Comment:
```suggestion
<user-details *ngFor="let user of (users | sortUsers: orderType.ASC)" [user]="user"
```
naming of the file isn't the same with the Enum